CHANGES.rst

```diff
@@ -6,7 +6,7 @@ Changelog
 
 Minor changes:
 
-- ...
+- The cli utility now displays start and end datetimes in the user's local timezone.
```

Comment:
Nice, could you reference the PR here and also your name?
Would you like to be part of the contributors' list?

Reply:
OK I referenced the PR in the changes file.

I don't think I need to be in the contributors' list for this small change, unless you really want me too.